### PR TITLE
updated domain in article-category #7653

### PR DESCRIPTION
**Pull request by amanpurohit**

### CODE REVIEW

File: tests/test_cms.py [hunk 1]
```diff
@@ -199,7 +199,7 @@ def test_0010_article_category(self):
             with app.test_client() as c:
                 response = c.get('/article-category/test-categ/')
                 self.assertEqual(response.status_code, 200)
-                self.assertEqual(response.data, '1')
+                self.assertEqual(response.data, '0')
```
Comment (tarunbhardwaj) [inline tests/test_cms.py]: please test with some published and draft articles

**Comment by PritishC** [comment]
LGTM

File: tests/test_cms.py (hunk 1 above)
Comment (tarunbhardwaj) [inline tests/test_cms.py]: this length should be 1

File: tests/test_cms.py [hunk 2]
```diff
@@ -194,12 +194,26 @@ def test_0090_article_states(self):
     def test_0010_article_category(self):
         "Successful rendering of an article_category page"
         with Transaction().start(DB_NAME, USER, CONTEXT):
+            article_categ1, = self.ArticleCategory.create([{
+                'title': 'Test Categ',
+                'unique_name': 'test-categ1',
+            }])
+            self.Article.create([{
```
Comment (tarunbhardwaj) [inline tests/test_cms.py]: use `test-categ` only and create published article in it.

File: tests/test_cms.py [hunk 3]
```diff
@@ -194,12 +194,26 @@ def test_0090_article_states(self):
...
+            }])
+
             self.setup_defaults()
             app = self.get_app()
             with app.test_client() as c:
-                response = c.get('/article-category/test-categ/')
+                response = c.get('/article-category/test-categ1/')
                 self.assertEqual(response.status_code, 200)
                 self.assertEqual(response.data, '1')
+                self.assertEqual(len(article_categ1.published_articles), 1)
```
Comment (tarunbhardwaj) [inline tests/test_cms.py]: assert total articles in category

File: tests/test_cms.py (hunk 2 above)
Comment (tarunbhardwaj) [inline tests/test_cms.py]: after setup_default

File: tests/test_cms.py [hunk 4]
```diff
@@ -196,10 +196,29 @@ def test_0010_article_category(self):
...
+
+            self.Article.create([{
+                'title': 'Test Article',
+                'uri': 'test-article1',
+                'content': 'Test Content',
+                'sequence': 10,
+                'categories': [('add', [self.article_categ.id])],
+                'state': 'published'
+            }])
+            self.Article.create([{
```
Comment (tarunbhardwaj) [inline tests/test_cms.py]: no need to call twice. can be done in same create

File: tests/test_cms.py [hunk 5]
```diff
@@ -196,10 +196,29 @@ def test_0010_article_category(self):
...
+
+            self.Article.create([{
+                'title': 'Test Article',
+                'uri': 'test-article1',
+                'content': 'Test Content',
+                'sequence': 10,
+                'categories': [('add', [self.article_categ.id])],
+                'state': 'published'
+            }, {
+                    'title': 'Test Article',
```
Comment (tarunbhardwaj) [inline tests/test_cms.py]: indentation

File: tests/test_cms.py [hunk 6]
```diff
@@ -196,10 +196,29 @@ def test_0010_article_category(self):
...
+                'categories': [('add', [self.article_categ.id])],
+                'state': 'published'
+            }, {
+                'title': 'Test Article',
+                'uri': 'test-article2',
+                'content': 'Test content',
+                'sequence': 20,
+                'categories': [('add', [self.article_categ.id])],
+                'state': 'draft'
+
```
Comment (tarunbhardwaj) [inline tests/test_cms.py]: remove this blank line

**Comment by tarunbhardwaj** [comment]
LGTM

**Pull request merged**